### DISCUSSION TITLE
multiple services same model

### DISCUSSION THREAD

**Discussion by devops724-2** [feathersjs/feathers#2366] [Q&A]
Hi there
setup two service and set model name as
const modelName = 'companies';
i try have same collection with multiple data design
but get this error on query 
"stack":"MongooseError: Operation `companies.countDocuments()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (/home/workspace/piotrek/dss/server/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:185:20)\n    at listOnTimeout (internal/timers.js:554:17)\n    at processTimers (internal/timers.js:497:7)","message":"Operation `companies.countDocuments()` buffering timed out after 10000ms"}] as {"type":3,"nsp":"/","id":3,"data":[{"stack":"MongooseError: Operation `companies.countDocuments()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (/home/workspace/piotrek/dss/server/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:185:20)\n    at listOnTimeout (internal/timers.js:554:17)\n    at processTimers (internal/timers.js:497:7)","message":"Operation `companies.countDocuments()` buffering timed out after 10000ms"}]} +10s

**Comment by crypto-titan** [comment]
+1
